### PR TITLE
DOC: Encourage trying preview release before reporting bug

### PR DESCRIPTION
**Pull request by lassoan**
Bug reporting checklist contained a note to test with the preview release, but many users would not take the effort to actually read and follow the instructions. Therefore it may make sense to emphasize this directly in the documentation.

### CODE REVIEW

File: Docs/user_guide/get_help.md [hunk 1]
```diff
@@ -10,7 +10,9 @@
 
 ## I want to report a problem
 
-If you are not sure if Slicer behaves incorrectly or you are not using it properly then [ask about it on the Slicer Forum](https://discourse.slicer.org/c/support) (in the `Support` category). If you are _sure_ that Slicer is not working as intended then [submit a bug report in the Slicer issue tracker](https://github.com/Slicer/Slicer/issues/new?assignees=&labels=type%3Abug&template=bug_report.md).
+We are constantly making fixes and improvements, therefore before you report any issues, check first if it is already fixed in the latest Slicer Preview Release.
+If you are not sure if Slicer behaves incorrectly or you are not using it properly then [ask about it on the Slicer Forum](https://discourse.slicer.org/c/support) (in the `Support` category).
```
Comment (pieper) [inline Docs/user_guide/get_help.md:14]:
```suggestion
If you find it hard to locate answers using the search feature of discourse or via internet searches such as Google, try a large language model like Chat-GPT, Gemini, etc.

If you are not sure if Slicer behaves incorrectly or you are not using it properly then [ask about it on the Slicer Forum](https://discourse.slicer.org/c/support) (in the `Support` category).
```

**Review by pieper — CHANGES_REQUESTED** [review]
It's good to beef this up.  Even is users never think to read this at least some AI might.

File: Docs/user_guide/get_help.md [hunk 2]
```diff
@@ -10,7 +10,9 @@
 
 ## I want to report a problem
 
-If you are not sure if Slicer behaves incorrectly or you are not using it properly then [ask about it on the Slicer Forum](https://discourse.slicer.org/c/support) (in the `Support` category). If you are _sure_ that Slicer is not working as intended then [submit a bug report in the Slicer issue tracker](https://github.com/Slicer/Slicer/issues/new?assignees=&labels=type%3Abug&template=bug_report.md).
+We are constantly making fixes and improvements, therefore before you report any issues, check first if it is already fixed in the latest Slicer Preview Release.
```
[the remark this answers is not in the document]
Comment (jcfr) [reply Docs/user_guide/get_help.md:13]:
```suggestion
We are constantly making fixes and improvements, therefore before you report any issues:
1. If you are using an older version, confirm the issue still exists in the current Stable release
2. If it still exists in the the stable version, confirm the issue still exists in the latest Preview version
```

**Comment by lassoan** [comment]
I've updated the text to use "Slicer Preview Release" and "Slicer Stable Release" consistently. I use these terms everywhere in documentation and forum discussions. Adding "Slicer" is important to make it clear that it is the Slicer core application (does not apply to custom applications or extensions). I use "Slicer" instead of "3D Slicer" to keep things shorter.

@pieper @jcfr We can switch to a different naming convention, but to avoid any confusion we should use the exact same words everywhere (rules around Slicer/extensions, Preview/Stable updates are already quite complex).

**Comment by pieper** [comment]
I've tried to avoid calling the preview a "release" to avoid confusion.  How about "Slicer Stable Release" and "Slicer Preview"

**Comment by lassoan** [comment]
We still need a word after "Slicer Preview". What would be that word?

"Release" captures the intent very accurately. It is not a version (there is a latest Slicer version at all times but a preview release borns when we actually release it as a package). Package could work, too but that too much focuses on the file itself, rather than the abstract object itself.

The word needs to pair well with "Slicer Stable Release", so all name suggestions should be reviewed together for the preview and stable.

- Slicer Stable Release / Slicer Preview => very odd, asymmetric
- Slicer Stable Release / Slicer Preview Version => Stable Release is a version, too
- Slicer Stable Release / Slicer Preview Package => release is a package, too

**Comment by pieper** [comment]
A "Preview" can be a thing, so Slicer Preview is grammatical. But maybe we should be more clear of the intent. Slicer Dev Preview or Slicer Beta Version would be better.

**Comment by lassoan** [comment]
Yes, maybe the issue is that we don't agree on the intent. For me, the preview release is a release. Intended for users, not developers. Maybe we can degrade the preview release once stable releases are created more often and the last two stable releases are supported (we cannot require users ro upgrade quarterly, sonekther we need LTS version or just always keep supporting the last two). 

We should find a term that is self-explaining, so words such as build, alpha, beta, etc. are not ideal.

**Comment by pieper** [comment]
I think everyone knows what a beta version is, so I don't see that as problematic.  But I do agree that our Previews are closer to Releases than for many projects.  For us I'd say it's rare for the Preview to be less "stable" than the Stable in the sense of bugs, but it's more stable in the sense of not changing frequently.  That's why I kind of miss the term Nightly.

**Comment by lassoan** [comment]
"Nightly" was confusing because the stable gets nightly updates, too.

I've searched on the web and found that while automatic updates made most of the users unaware about software versioning, the "beta" term is still used in developer/insider programs. So, developers and early adopters should be familiar with this term.

Interestingly, [Microsoft distinguishes beta releases and preview releases](https://insider.microsoft365.com/en-us/join/windows#2.-Choose-an-Insider-level). Beta is less stable, not supported; preview is for early adopters, fully supported - which exactly matches our current nomenclature.

**Comment by jamesobutler** [comment]
GitHub uses “Release” for stable tags while “Pre-release” is used for release candidates, beta builds, alpha builds etc.

Chrome has the following channels:
Stable -> Beta -> Dev -> Canary
https://www.chromium.org/getting-involved/dev-channel/

The term for the non-stable build probably doesn’t matter a whole lot as it seems like most users will stick to just using the stable build unless they are a developer. As most users are not early adopters. I’m not sure you’re going to be able to change that behavior. Based on https://download.slicer.org/download-stats/, in the past year 77% of downloads have been of a stable release and when looking at just the United States that goes up to 83%. I think most (not all) users will consider a bug present until the fix makes it into a stable release. How inherently well Slicer works is going to be judged by the Stable release.

**Comment by jcfr** [comment]
The terms `Preview` and `Stable` release make sense and using these consistently is sensible.

I suggest we revisit the wording independently of this pull request. This is more larger task that would require update to not only the documentation but also `slicer.org` and  `download.slicer.org`.

**Review by pieper — APPROVED** [review]
Yes, let's move ahead with this.

**Pull request merged**